ref-exchange/src/errors.rs

```diff
@@ -0,0 +1,12 @@
+// Storage errors //
```

Comment:
let's not use // at the end
/// is for doc strings
//! for file doc strings

Reply:
OK. BTW: this is not a doc string.